### PR TITLE
Fixes roulette exploit.

### LINKED ISSUE #61126: how to beat the house at roulette
betting on evens gives the player better odds of winning than the house gets, as rolling "0" counts as rolling an even number (unlike in actual roulette, which has rolling "0" give the win to the house regardless of whether you bet evens or odds).

### PR DESCRIPTION
**Pull request by Pickle-Coding**
## About The Pull Request
Fixes #61126 

## Why It's Good For The Game
Exploit bad.

## Changelog

:cl: AccountName5
fix: Roulette: You no longer win when betting on even and rolling 0.
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
What happens if you choose zero? cause it seems that you'll loose everytime even if you choose zero and it outputs zero

**Comment by 170141183460469231731687303715884105727** [comment]
Is there any reason why the statement above the switch is if(actual_bet_number)?

File: code/game/machinery/roulette_machine.dm [hunk 1]
```diff
@@ -291,7 +291,7 @@
 		if(ROULETTE_BET_ODD)
 			return ISODD(rolled_number)
```
Comment (Ghommie) [inline code/game/machinery/roulette_machine.dm:292]:
```suggestion
			return (ISODD(rolled_number) && rolled_number != 0)
```
0 and 00 are excluded from even/odd bets. The house always wins when they're rolled.

**Comment by 170141183460469231731687303715884105727** [comment]
> What happens if you choose zero? cause it seems that you'll loose everytime even if you choose zero and it outputs zero

Could I do this? `if(actual_bet_number || rolled_number == 0)`

**Review by Pickle-Coding — COMMENTED** [review]
Someone deleted their review before I replied, causing me to somehow accidentally send a review. I guess I'll send this and find out how to delete this useless review.

**Pull request merged**